src/matrix-utils.ts

```diff
@@ -269,11 +272,17 @@ export function isLocalRoomId(roomId: string, client: MatrixClient): boolean {
   return parts[1] === client.getDomain();
 }
 
+interface CreateRoomResult {
```

Comment:
In other instances its called the other way around: CreateRoomResult vs roomCreateResult. Is this intentional?

Reply:
Mmm, this probably could be more consistent - fixed